index.js

```diff
@@ -0,0 +1,53 @@
+let bookCollection = localStorage.getItem('bookCollection')
+  ? JSON.parse(localStorage.getItem('bookCollection'))
+  : [];
+const title = document.querySelector('#title-input');
+const author = document.querySelector('#author-input');
+const bookContainer = document.querySelector('.books-container');
+
+const removeBook = function () {
+  const title = this.id.split('-')[0];
+  const author = this.id.split('-')[1];
+  bookCollection = bookCollection.filter(
+    (book) => book.title !== title && book.author !== author,
+  );
+
+  displayBooks(); // eslint-disable-line no-use-before-define
```

Comment:
- [ ] Linters are here to help you write a better code, if this can be done on another way please remove the disabling and update the code if necessary